resources/views/entities/pages/attribute-templates/_form.blade.php

```diff
@@ -22,15 +21,11 @@
     'marketplace' => link_to(config('marketplace.url') . '/attribute-templates', __('front.menu.marketplace'), ['target' => '_blank'])
     ]) !!}
         </p>
-
-        <div class="my-5 text-center">
-            <button type="button" class="btn btn-default mr-5 rounded-full px-8" data-dismiss="modal">
-                {{ __('crud.cancel') }}
-            </button>
-
-            <button class="btn btn-success ml-5 rounded-full px-8">
-                {{ __('crud.actions.apply') }}
-            </button>
+        <div class="pull-right">
```

Comment:
Please keep the old "centered buttons" look to keep it consistent with the other round-button popups 
![image](https://user-images.githubusercontent.com/3263440/206585175-b92d905a-7da4-40f4-a8d9-336ea183c967.png)
Hint: you'll want to add `inline-block` as a class to the `submit-group` div.